source/documents/insert.txt

```diff
@@ -52,6 +52,12 @@ field name and value.
           "phone": "555-555-1212"
         }
 
+Limitation
+----------
+
+The :guilabel:`Insert Document` button is not available if you are connected 
+to a :atlas:`Data lake </data-lake>`.
```

Comment:
@kanchana-mongodb Can you submit another PR to capitalize "lake" ?

"Data Lake"